test/e2e/alcotest/failing/check_basic.expected

```diff
@@ -1,24 +1,160 @@
- ...                different composite          1   array.
[FAIL]              different composite          1   array.
- ...                different composite          2   option some.
[FAIL]              different composite          2   option some.
- ...                different composite          3   result.
[FAIL]              different composite          3   result.
- ...                different composite          4   pair.
[FAIL]              different composite          4   pair.
--- different basic.000 [bool.] Failed --
-in `<build-context>/_build/_tests/<uuid>/different basic.000.output`:
-ASSERT bool
-[failure] Error bool: expecting
-true, got
-false.
-
-The full test results are available in `<build-context>/_build/_tests/<uuid>`.
-13 errors! in <test-duration>s. 13 tests run.
+Testing `check_basic'.
+This run has ID `<uuid>'.
+
+  ...           different basic              0   bool.ASSERT bool
+
  [FAIL]        different basic              0   bool.
+  ...           different basic              1   int.ASSERT int
+
  [FAIL]        different basic              1   int.
+  ...           different basic              2   int32.ASSERT int32
+
  [FAIL]        different basic              2   int32.
+  ...           different basic              3   int64.ASSERT int64
+
  [FAIL]        different basic              3   int64.
+  ...           different basic              4   float.ASSERT float
+
  [FAIL]        different basic              4   float.
+  ...           different basic              5   char.ASSERT char
+
  [FAIL]        different basic              5   char.
+  ...           different basic              6   string.ASSERT string
+
  [FAIL]        different basic              6   string.
+  ...           different basic              7   bytes.ASSERT bytes
+
  [FAIL]        different basic              7   bytes.
+  ...           different composite          0   list.ASSERT list
+
  [FAIL]        different composite          0   list.
+  ...           different composite          1   array.ASSERT array
+
  [FAIL]        different composite          1   array.
+  ...           different composite          2   option some.ASSERT option some
+
  [FAIL]        different composite          2   option some.
+  ...           different composite          3   result.ASSERT result
+
  [FAIL]        different composite          3   result.
+  ...           different composite          4   pair.ASSERT pair
+
  [FAIL]        different composite          4   pair.
+
+┌──────────────────────────────────────────────────────────────────────────────┐
+│ [FAIL]        different basic              0   bool.                         │
+└──────────────────────────────────────────────────────────────────────────────┘
+FAIL bool
+
+   Expected: true
+   Received: false
+ ──────────────────────────────────────────────────────────────────────────────
+
+
+┌──────────────────────────────────────────────────────────────────────────────┐
```

Comment:
Is the new default behaviour to show all the failures? I don't think that's a good idea, the previous default is just to show the first one unless `ALCOTEST_SHOW_ERRORS` is set (if I remember correctly).

Reply:
No, this isn't default. There's a corresponding change to the `Alcotest.run` to pass `~verbose:true` (https://github.com/mirage/alcotest/pull/246/files#diff-fb43ad350d97f3cf0e25d21ec53a881dR7).

I felt that this is a better test if all of the errors are printed.